draft-ietf-quic-transport.md

```diff
@@ -3517,6 +3517,13 @@ use the following steps on receiving an ACK frame to validate ECN.
   corresponding ECT codepoint that are newly acknowledged in this ACK frame.
   This step detects any erroneous network remarking from ECT(0) to ECT(1) (or
   vice versa).
+  
+Another special case is when all packets are indicatd as CE marked,
+independtly of the initial codepode used, including Not-ECT. In this case
+it could be a network element overwriting the ECN field of all packets and
```

Comment:
Let's not say why we think this is happening and go straight to the action:

> If a peer indicates that all received packets are ECN-CE marked, endpoints MAY treat this as an ECN validation failure and ignore any ECN-CE counts in acknowledgments.

Reply:
I theory this could also be a valid congestion signal. Even if packets were marked Not-ECT there could one element that overwrite it to ECT and another later on that is actually congested and marks validly all packets as CE (I know it's rathe unlikely, just saying...). I found it kind of important to say that this is problematic as it could also conceal valid signals. How about:

"If this is caused by a network element erroneously overwriting the ECN code point field, it will also conceals actual congestion information. Therefore, if a peer indicates that all received packets are ECN-CE marked, endpoints MAY treat this as an ECN validation failure and ignore any ECN-CE counts in acknowledgments."

Reply:
WFM s/conceals/conceal/